jsonnet/kube-prometheus/kube-prometheus-custom-metrics.libsonnet

```diff
@@ -0,0 +1,179 @@
+local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
```

Comment:
Do we not want to also generate the files coming from this as an example users can try out, same way as we had it before? @brancz ?

Reply:
The feedback was that custom-metrics aren't fully ready yet and so this addon should only be opt-in vs. installed by default https://github.com/coreos/kube-prometheus/pull/374#issuecomment-573692256

Reply:
Right now we are removing the example experimental directory, all I would want is the files from this libsonnet to be generated by default same in the same way as they were before in the experimental directory. I agree it should be opt-in, but this just makes it harder for folks to try out.

Reply:
Make sense, I added a new commit to generate the experimental manifest from jsonnet.

Reply:
We don't generate any of the other additive things like kops, kubespray, eks, aks, gke, ... I don't think we should start doing this for any other add-ons as it sets the wrong precedence in my opinion. We should focus on driving people towards generating in my opinion.

Reply:
Happy to remove the 2nd commit again, if that is the agreement :-)

Reply:
Sorry about that! 🤷 up to frederic I guess, we discussed this OOB and feel free to remove the commit! 

But @brancz needs to open an issue about testing the manifests we ship or at least producing the manifests as part of CI runs so we can inspect any problems.

Reply:
@lilic no worries, removed :-)